### ISSUE TITLE
a_value method signature: a_value(y_true, y_pred_prob, zero_label=0, one_label=1):

### ISSUE THREAD

**Issue by arilwan** [pritomsaha/Multiclass_AUC#1]
Just curious to ask, why do you set `zero_label=0` , `one_label=1`?

I noticed if you have `multiclass` in which classes are not label `0, 1,,...` you run into `zero division` thus final `mauc`  value is `nan`.

Calling the method `a_value()`

Step to reproduce:

```
# define dataset
X, y = make_classification(n_samples=1000, n_features=3, n_redundant=0, n_classes=3,
	n_clusters_per_class=1, weights=[0.7, .2, .1], flip_y=0, random_state=1,)#  hypercube=False)

from sklearn.ensemble import RandomForestClassifier
from sklearn.model_selection import train_test_split 

xtrain, xtest, ytrain, ytest = train_test_split(X, y, test_size=.2, random_state=2)

clf = RandomForestClassifier(random_state=1)
clf.fit(xtrain, ytrain)

ypred = clf.predict_proba(xtest)

mauc = MAUC(ytest, ypred, 3)
mauc
0.9196769014649809
```
Now:
```
ytrain[ytrain == 0] = 9
ytest[ytest == 0] = 9

np.unique(ytrain)
array([1, 2, 9])

clf.fit(xtrain, ytrain)
ypred = clf.predict_proba(xtest)

mauc = MAUC(ytest, ypred, 3)
RuntimeWarning: invalid value encountered in double_scalars
  sum_ranks = np.sum(np.where(sorted_ranks==zero_label)) + n0

print(mauc)
nan
```
Does that mean implementation calculates mauc for `class 0` and `1` ignoring other classes?

**Comment by pritomsaha** [comment]
Please see the note-
NB: The class labels should be in the set [0,n-1] where n = # of classes.
    The class probability should be at the index of its label in the
    probability list.

Regarding "Just curious to ask, why do you set zero_label=0 , one_label=1?" I did not set zero_label=0 , one_label=1, zero_label is just the first element in the class pair and one_label is the second element.

For, classes {0,1,2}, corresponding zero and one lables are:
(0, 1)
(0, 2)
(1, 2)
 
For further reference, see Hand and Till's 2001 paper. Hope it will help.

**Comment by arilwan** [comment]
Ah, I understand, 

Thank you.
